intro_to_R_tidyverse/01-intro_to_r.Rmd

```diff
@@ -344,7 +337,7 @@ not_sq_matrix <- matrix(1:100)
 str(not_sq_matrix)
```

Comment:
Can we introduce `dim` and use it here? Also `nrow` and `ncol`

Reply:
I'll add `dim`, but I don't want to overload them with every function at once. Plus dim gives both anyway.